Rubberduck.Parsing/ComReflection/ComModule.cs

```diff
@@ -52,7 +53,7 @@ private void GetComFields(ITypeInfo info, TYPEATTR attrib)
                 info.GetNames(desc.memid, names, names.Length, out int length);
                 Debug.Assert(length == 1);
 
-                _fields.Add(new ComField(info, names[0], desc, index, DeclarationType.Constant));
+                _fields.Add(new ComField(info, desc, names[0], index, DeclarationType.Constant));
                 info.ReleaseVarDesc(varPtr);
```

Comment:
Potential to leak pointer on an exception. Should use dispoal action container.

Reply:
Damnit! I thought I found all of those. Good spot.